src/components/leftSection/LeftSection.jsx

```diff
@@ -33,104 +32,46 @@ function LeftSection() {
         <nav>
           <div className="top">
             <div className="links">
-              <ul>
-                <li
-                  onClick={() => setCurrentLink(1)}
-                  className={currentLink === 1 ? "active" : ""}
-                >
-                  <a href="#">
-                    <MdOutlineHome />
-                    <span>Home</span>
-                  </a>
-                </li>
-                <li
-                  onClick={() => setCurrentLink(2)}
-                  className={currentLink === 2 ? "active" : ""}
-                >
-                  <a href="#">
-                    <MdOutlineDashboard />
-                    <span>Dashboard</span>
-                  </a>
-                </li>
-                <li
-                  onClick={() => setCurrentLink(3)}
-                  className={currentLink === 3 ? "active" : ""}
-                >
-                  <a href="#">
-                    <MdOutlineAccountBalanceWallet />
-                    <span>Wallet</span>
-                  </a>
-                </li>
-                <li
-                  onClick={() => setCurrentLink(4)}
-                  className={currentLink === 4 ? "active" : ""}
-                >
-                  <a href="#">
-                    <MdOutlineNotificationsNone />
-                    <span>Notification</span>
-                  </a>
-                </li>
-                <li
-                  onClick={() => setCurrentLink(5)}
-                  className={currentLink === 5 ? "active" : ""}
-                >
-                  <a href="#">
-                    <MdOutlineQuestionAnswer />
-                    <span>FAQ</span>
-                  </a>
-                </li>
-                <li
-                  onClick={() => setCurrentLink(6)}
-                  className={currentLink === 6 ? "active" : ""}
-                >
-                  <a href="#">
-                    <FaBlog />
-                    <span>Blog</span>
-                  </a>
-                </li>
-              </ul>
+              <NavLink to="/dashboard" className="nav-link">
```

Comment:
This whole block of code is repetitive so you can create a separate component or function for this and pass props here